src/examples/nintendoswitch/debug-hello.go

```diff
@@ -0,0 +1,23 @@
+// This program is a minimal Nintendo Switch program that just outputs messages to
+// emulator console
+package main
+
+import (
+	"runtime"
+	"time"
+)
+
+func print(data string) {
```

Comment:
Try renaming this to avoid overloading the language builtin of the same name. Maybe `debugPrint`?